yarn.lock

```diff
@@ -3166,6 +4733,28 @@ ember-cli-broccoli-sane-watcher@^3.0.0:
     rsvp "^3.0.18"
     sane "^4.0.0"
 
+ember-cli-clipboard@^0.11.1:
+  version "0.11.1"
+  resolved "https://registry.yarnpkg.com/ember-cli-clipboard/-/ember-cli-clipboard-0.11.1.tgz#caa6aaae498f12922102555d6825ad81ad843d2a"
+  integrity sha512-LAsrFpaOV8mgyI4MLR6R2BJFbzW8ac3GZkTKmAH+V5LeyidK/Inr4yZpY5nff/jKlQFT6E4991Z9mzldfKJZcg==
+  dependencies:
+    broccoli-funnel "^1.1.0"
+    clipboard "^2.0.0"
+    ember-cli-babel "^7.1.2"
+    ember-cli-htmlbars "^2.0.2"
+    fastboot-transform "0.1.1"
+
+ember-cli-clipboard@^0.8.1:
```

Comment:
duplicate `ember-cli-clipboard` dependency 🤔

Reply:
Odd. Just ran `yarn-deduplicate`. Pushing that in now.